From 5434d492b7fd27d551742c1d735634115e0b0e9a Mon Sep 17 00:00:00 2001
From: Peng Fan <peng.fan@nxp.com>
Date: Mon, 13 Nov 2017 09:05:34 +0800
Subject: [PATCH 2782/5242] MLK-16803 arm64: dts: imx8mq-evk: removed the
 reserved ATF space

commit  6d8a3d31f1a999d6f9f34cde4b5e7d90147f7ab2 from
https://source.codeaurora.org/external/imx/linux-imx.git

ATF has been moved to OCRAM, there is no need to reserve space
for ATF in DRAM space.

Signed-off-by: Peng Fan <peng.fan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |    3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index 1675216..8b5151e 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -15,9 +15,6 @@
 
 /dts-v1/;
 
-/* First 128KB is for PSCI ATF. */
-/memreserve/ 0x40000000 0x00020000;
-
 #include "fsl-imx8mq.dtsi"
 
 / {
-- 
1.7.9.5

